api/client/service.proto

```diff
@@ -30,11 +30,8 @@ message ExecutionRequest {
 }
 
 message ExecutionResponse {
-  oneof status_specifier {
-    option (validate.required) = true;
-    Output output = 1;
-    google.rpc.Status error_detail = 7;
-  }
+  Output output = 1;
```

Comment:
FWIW, these breaking changes are going to have to not be allowed as soon as Google or some external is making use of Nighthawk in any CI etc.

Reply:
Ack; AFAICT this didn't break compilation, but it's a behavioral change nonetheless.